scripts/generate-ckpt.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+set -e
+
+usage() {
+    echo "Usage: $0 [OPTIONS] -- [SPIKEFLAGS]"
+    echo ""
+    echo "Options"
+    echo "  --help -h  : Display this message"
+    echo "  -n <n>     : Mumber of harts"
+    echo "  -b <elf>   : Binary to run in spike"
+    echo "  -p <pc>    : PC to take checkpoint at [default 0x80000000]"
```

Comment:
If you want the checkpoint to be at a certain PC, you can do `-p <pc>`. If you want the checkpoint to be a certain number of instructions after the start of the binary, you can just do `-c <cycles>`. 

The default is 0x80000000 because that is where _start usually is in our binaries.